app/com/arpnetworking/kairos/client/models/Aggregator.java

```diff
@@ -78,6 +80,40 @@ public String getName() {
         return _otherArgs;
     }
 
+    @Override
+    public String toString() {
+        return MoreObjects.toStringHelper(this)
+                .add("_name", _name)
```

Comment:
nit: (it's gotten inconsistent but) preference is to use field names with the underscore prefix for the name.